src/mongo/db/repl/topology_coordinator_impl.cpp

```diff
@@ -1539,6 +1539,7 @@ void TopologyCoordinatorImpl::prepareStatusResponse(const ReplSetStatusArgs& rsS
             BSONObjBuilder bb;
             bb.append("_id", _selfConfig().getId());
             bb.append("name", _selfConfig().getHostAndPort().toString());
+            bb.append("internal", _selfConfig().getHostInternalAndPort().toString());
```

Comment:
Also this should probably be "getInternalHostAndPort()".  Unless you mean that the port isn't part of the internalness.